react/components/SKUSelector/components/SelectorItem.js

```diff
@@ -42,7 +44,7 @@ export default class SelectorItem extends PureComponent {
           <div
             className={classNames(
               'absolute absolute--fill',
-              isAvailable || 'diagonal-cross'
+              isAvailable || `${styles.diagonalCross}`
```

Comment:
This seems weird to me, what happens when isAvailable is true? This expression is just returning a boolean then, right?

Reply:
When it's a `||` it returns the last truty value (from left to right).
However, I think it's better to fix and use the classNames syntax with objects.

Reply:
Yeah, but `isAvailable` is a boolean, so when its value is truthy it will return it, so it will return `true`, but you're changing it, so whatever :100:

Reply:
Yep, It's a Boolen, if it is `true` it will return the value in `styles.diagonalCross` if it is truthy, if it is falsy will return `true`. I think someone confused `||` with `&&` when made this line of code.

Reply:
Wow, I'm totally wrong. `||` operator returns the first truthy value.